### PR TITLE
Add TC gluster_volume_profiling

### PR DESCRIPTION
**Pull request by dahorak**

### CODE REVIEW

**Review by ltrilety — CHANGES_REQUESTED** [review]
Wide volume profiling enables/disables volume profiling for all volumes existing and new ones. So the test should be made with this bearing in mind.
Moreover to check if volume profiling is enabled there has to be some volume in gluster. There should be some mention about that in test set-up.

File: web/tendrl/gluster_volume_profiling.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+************************
+
+:authors: 
+          - dahorak@redhat.com
+
+Description
+===========
+
+Enable/disable cluster wide volume profiling 
+(both on initial import and post import).
```
Comment (mbukatov) [inline web/tendrl/gluster_volume_profiling.rst]: Could you link upstream docs for this feature?

**Review by ltrilety — APPROVED** [review]
LGTM

**Review by mbukatov — CHANGES_REQUESTED** [review]
This seems to need updates. It would be also nice to provide references to upstream docs/requirements.

**Comment by mbukatov** [comment]
Clarification pending.

**Review by mbukatov — APPROVED** [review]
Looks good.

**Pull request merged**